tests/testthat/test-as_flex_table.R

```diff
@@ -303,22 +303,22 @@ test_that("as_flex_table passes multiple table footnotes correctly", {
     modify_table_styling(
       columns = stat_1,
       rows = (variable %in% "grade") & (row_type == "level"),
-      footnote = "Cell-level foonotes here."
+      footnote = "my footnote"
```

Comment:
why?

Reply:
the bug appeared when we were assigning the same footnote to multiple locations. separate footnotes were assigned with separate calls to the flextable footnote function

Reply:
I see! thanks